src/fidesops/core/config.py

```diff
@@ -191,6 +191,14 @@ class Config:
         env_prefix = "FIDESOPS__SECURITY__"
 
 
+class RootUserSettings(FidesSettings):
+    """Configuration settings for Analytics variables."""
+
+    ANALYTICS_OPT_OUT: Optional[bool]
+    # analytics id should be set to "internal" when in development / testing
```

Comment:
This should only be documented internally. As an open-source repo, we risk having random non-Ethyca employee users set their `ANALYTICS_ID` to `"internal"`, and skew our analytics.

Unfortunately I don't know the best place to document this. Previously I just reminded people to set the variable in their local ENV's a few times via Slack 🤷‍♂️

Reply:
Good point, removed!